test/healthcheck/framework/metrics.go

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package metrics creates and registers metrics objects with Prometheus
+// and sets the Prometheus HTTP handler for /metrics
+package framework
+
+import (
+	"net/http"
+	"sync"
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+var registerMetrics sync.Once
+
+const (
+	healthcheckNamespace = "catalog_health" // Prometheus namespace (nothing to do with k8s namespace)
```

Comment:
Maybe call this `prometheusNamespace` then? Would help if someone encounters the variable name in the code below (and thus doesn't see this comment)